test/remote/gateways/remote_allied_wallet_test.rb

```diff
@@ -32,11 +32,12 @@ def test_failed_purchase_no_address
   end
 
   def test_successful_purchase_with_more_options
-    response = @gateway.purchase(@amount, @credit_card, @options.merge(
+    options = @options.merge(
       order_id: generate_unique_id,
       ip: '127.0.0.1',
       email: 'jim_smith@example.com'
-    ))
+    )
+    response = @gateway.purchase(@amount, @credit_card, options)
```

Comment:
I received 14 errors for this remote test file on both my branch and on master. This is not a Spreedly supported gateway integration, so we don't appear to have the correct credentials.